styles/colors-silicon-day.less

```diff
@@ -298,6 +302,17 @@
 @agate-switch-focus-color: black;
```

Comment:
Please do not use CSS color value directly. You could use @agate-black instead of black.